src/analytics/src/main/scala/osmesa/analytics/oneoffs/StreamingAOIMonitor.scala

```diff
@@ -1,234 +1,236 @@
-      )
-      .orElse(Opts.env[URI]("DATABASE_URL", help = "The URL of the database"))
-
-    val augmentedDiffSourceOpt =
-      Opts
-        .option[URI](
-        "augmented-diff-source",
-        short = "a",
-        metavar = "uri",
-        help = "Location of augmented diffs to process"
-      )
-
-    val startSequenceOpt =
-      Opts
-        .option[Int](
-        "start-sequence",
-        short = "s",
-        metavar = "sequence",
-        help = "Starting sequence. If absent, the current (remote) sequence will be used."
-      )
-      .orNone
-
-    val endSequenceOpt =
-      Opts
-        .option[Int](
-        "end-sequence",
-        short = "e",
-        metavar = "sequence",
-        help = "Ending sequence. If absent, this will be an infinite stream."
-      )
-      .orNone
-
-    (intervalOpt, databaseUrlOpt, augmentedDiffSourceOpt, startSequenceOpt, endSequenceOpt).mapN {
-      (interval, databaseUri, augmentedDiffSource, startSequence, endSequence) =>
-      val appName = "StreamingAOIMonitor"
-
-      implicit val spark: SparkSession = Analytics.sparkSession("Streaming AOI Monitor")
-
-      import spark.implicits._
-      spark.withJTS
-      import AOIMonitorUtils._
+object StreamingAOIMonitor
+    extends CommandApp(
+      name = "streaming-aoi-monitor",
+      header = "Streaming AOI Monitor",
+      main = {
+        val intervalOpt =
+          Opts
+            .option[Interval]("interval",
+                              help = "Period of time to aggregate over (d=daily, w=weekly)")
+
+        val augmentedDiffSourceOpt =
+          Opts
+            .option[URI](
+              "augmented-diff-source",
+              short = "a",
+              metavar = "uri",
+              help = "Location of augmented diffs to process"
+            )
+
+        val startSequenceOpt =
+          Opts
+            .option[Int](
+              "start-sequence",
+              short = "s",
+              metavar = "sequence",
+              help = "Starting sequence. If absent, the current (remote) sequence will be used."
+            )
+            .orNone
+
+        val endSequenceOpt =
+          Opts
+            .option[Int](
+              "end-sequence",
+              short = "e",
+              metavar = "sequence",
+              help = "Ending sequence. If absent, this will be an infinite stream."
+            )
+            .orNone
+
+        (intervalOpt, augmentedDiffSourceOpt, startSequenceOpt, endSequenceOpt).mapN {
+          (interval, augmentedDiffSource, startSequence, endSequence) =>
+            val appName = "StreamingAOIMonitor"
+
+            implicit val spark: SparkSession = Analytics.sparkSession("Streaming AOI Monitor")
+
+            import spark.implicits._
+            spark.withJTS
+            import AOIMonitorUtils._
+
+            val areasOfInterest: List[AOIData] = queryAreasOfInterest()
+            areasOfInterest.foreach { println(_) }
```

Comment:
With the non-compiling section below commented out, I ran the following:
```
 test:runMain osmesa.analytics.oneoffs.StreamingAOIMonitor --interval w --augmented-diff-source s3://<bucket>/augmented-diffs
```
and got:
```
[info] AOIData(2756a4a0-08ce-46de-aee9-8959a2548777,0587df26-28b5-4374-a9ea-8a4cf2de2d3f,2019-10-08 17:20:34.574671,POLYGON ((-67.32421875 9.1
78024574578373, -65.511474609375 9.178024574578373, -65.511474609375 10.746969318460001, -67.32421875 10.746969318460001, -67.32421875 9.17802
4574578373)),Caracas,foo@example.com) 
```
printed from this line which connected to the database holding this information.